### DISCUSSION TITLE
How to make two supervisors line-wrapped and displayed in alignment？

### DISCUSSION THREAD

**Discussion by nwangxy** [ustctug/ustcthesis#393] [Q&A]
I'd like to add two professors and want them to appear on different lines and aligned
supervisor：Prof.xx 
                    Prof. xxx

Please guide me, I will be very grateful to you
regards.

**Comment by zepinglee** [comment]
I'm afraid this is not possible to implement in a trivial way. You need to modify the relevant part of the template.

https://github.com/ustctug/ustcthesis/blob/4e5dad905a545801b182a37995f5aa30a7380795/ustcthesis.cls#L1902-L1912
